ArjunaCore/arjuna/classes/com/arjuna/ats/internal/arjuna/objectstore/hornetq/HornetqJournalEnvironmentBean.java

```diff
@@ -40,6 +40,8 @@
 
     private volatile int minFiles = 4;
 
+    private volatile int poolSize = -1;
```

Comment:
The poolSize featured was add with https://issues.apache.org/jira/browse/ARTEMIS-319

From the documentation: 
```
The upper theshold of the journal file pool,-1 (default) means no Limit. 
The system will create as many files as needed however when reclaiming files
it will shrink back to the `journal-pool-files`
```
(I guess they meant `journal-min-files` there) and also:
```
Set `journal-pool-files` to a number that represents
something near your maximum expected load.  The journal will spill over
the pool should it need to, but will shrink back to the upper threshold, 
when possible.  This allows reuse of files, without taking up more disk
space than required.  If you see new files being created on the journal
data directory too often, i.e. lots of data is being persisted, 
you need to increase the journal-pool-size, this way the journal would
reuse more files instead of creating new data files, increasing performance
```